sklearn/covariance/empirical_covariance_.py

```diff
@@ -69,7 +69,8 @@ def empirical_covariance(X, assume_centered=False):
     """
     X = np.asarray(X)
     if X.ndim == 1:
-        X = np.reshape(X, (1, -1))
+        X = X[:, np.newaxis]
```

Comment:
`check_array` does raise a warning, but tests fail because somewhere else a 2d array is expected ( I don't remember where in this case )
I assumed a 1D array implied data with a single feature, that's why I kept rows same and made sure it had one column.